engine-precompiles/src/native.rs

```diff
@@ -326,6 +327,10 @@ impl<I: IO> Precompile for ExitToNear<I> {
                 let amount = U256::from_big_endian(&input[..32]);
                 input = &input[32..];
 
+                if amount > MAX_U128.into() {
```

Comment:
```suggestion
                if amount > U256::from(u128::MAX) {
```
Something along those lines above. Avoid use of `.into`. This will be banned with checks in the future.